lib/suspenders/app_builder.rb

```diff
@@ -19,6 +19,23 @@ def set_test_delivery_method
       )
     end
 
+    def add_bullet_gem_configuration
+      config = <<-RUBY
+  config.after_initialize do
+    Bullet.enable = true
+    Bullet.bullet_logger = true
+    Bullet.rails_logger = true
+  end
+
+      RUBY
+
+      inject_into_file(
+        'config/environments/development.rb',
```

Comment:
Prefer double-quoted strings unless you need single quotes to avoid extra backslashes for escaping.